ocaml/typing/layouts.ml

```diff
@@ -323,6 +323,7 @@ module Layout = struct
     | Type_wildcard of Location.t
 
   type creation_reason =
+    | Err_msg_attr of string
```

Comment:
Often better to avoid abbreviations: code is read *much* more than it is written.